src/ducks/balance/components/AccountRow.jsx

```diff
@@ -84,9 +86,16 @@ class AccountRow extends React.PureComponent {
       disabled,
       onSwitchChange,
       id,
-      triggerCol
+      triggerCol,
+      showOwners,
+      client
     } = this.props
 
+    const contacts = client.getCollectionFromState(CONTACT_DOCTYPE)
+    const contactsById = keyBy(contacts, contact => contact._id)
+    const ownerRelationships = get(account, 'relationships.owners.data', [])
+    const owners = ownerRelationships.map(data => contactsById[data._id])
```

Comment:
Nit: IMHO a selector would be better as it would memoize more easily.

Reply:
I added a todo comment